pcdsdevices/gon.py

```diff
@@ -323,13 +345,19 @@ def __init__(self, *, name, prefix_x, prefix_y, prefix_z,
         self.kappa_max_step = kappa_max_step
         self.phi_max_step = phi_max_step
         self.kappa_ang = kappa_ang
+
```

Comment:
```suggestion
        self._prefix_x = prefix_x
        self._prefix_y = prefix_y
        self._prefix_z = prefix_z
```

Reply:
I have these guys already :) - unless you purposely suggest to move them further down